### ISSUE TITLE
PC crashes in post game

### ISSUE THREAD

**Issue by jojobear13** [jojo-kep/kep-hack#7]
![image](https://github.com/jojo-kep/kep-hack/assets/47074297/0254eb23-dc6c-46b1-abcf-86357b916bfc)

**Comment by jojobear13** [comment]
[kep.zip](https://github.com/jojo-kep/kep-hack/files/14320231/kep.zip)

**Comment by jojobear13** [comment]
**Root Cause**

KEP changes the OakLabEmailText from a piece of hidden object text (like a bookshelf) into the kind of background-based text used for overworld signs. OakLabEmailText then becomes the 30th entry of OaksLab_TextPointers.

The problem is that Oak's lab has strange vanilla scripting where getting the free pokeballs changes the text script pointer table for the map.

```
	CheckEvent EVENT_PALLET_AFTER_GETTING_POKEBALLS_2
	call nz, OaksLabScript_1d076
```
```
OaksLabScript_1d076:
	ld hl, OaksLab_TextPointers2
	ld a, l
	ld [wMapTextPtr], a
	ld a, h
	ld [wMapTextPtr + 1], a
	ret
```

Because OaksLab_TextPointers2 does not have a 30th entry, trying to use the PC once the event bit has been set will cause garbage data to be sent to the text command function.

**Comment by jojobear13** [comment]
Fixed with commit 65c8f70856670bb5e832850ae064a19653da835a
